templates/CONFIG_TEMPLATE.INI

```diff
@@ -360,13 +357,7 @@
 
 #  sets the space token for storing temporary files (e.g. oversized input
 #  sandbox)
-#DefaultSRMToken = 
-
-#  enables/disables the support of the EDG middleware
-#EDG_ENABLE = True
-
-#  sets the LCG-UI environment setup script for the EDG middleware
-#EDG_SETUP = /afs/cern.ch/sw/ganga/install/config/grid_env_auto.sh
+#DefaultSRMToken =
```

Comment:
In that case there is no hard updating them here and we can burn them in a separate PR